easy-ui-react/src/Icon/Icon.module.scss

```diff
@@ -7,10 +7,12 @@
     height: $var;
   }
 
-  display: block;
+  display: flex;
```

Comment:
fixes some cases where the icon will incorrectly mess up the parent container's layout behavior

Reply:
i had been seeing some folks using an additional wrapper around icons, this addresses that, nice!